internal/docker/container.go

```diff
@@ -262,10 +263,27 @@ func (r *ContainerRunner) runJobs(containerOpts <-chan containerStartOptions, re
 		}
 		start := time.Now()
 		containerID, output, jobDetails, passed, skipped, err := r.runSuite(opts)
+
+		browser := opts.Browser
+		// Playwright uses its own bundled browsers, not the ones installed in the image
+		if r.Framework.Name != "playwright" {
```

Comment:
Maybe it's better to create a separate function for getting a playwright browser?

Reply:
Getting the playwright browser version(s) will end up being similar to this implementation, i.e. reading from a label on the image. The playwright image doesn't currently have the correct browser versions in its label metadata so I've opted to skip it for now.

Reply:
I mean that `if.. else if` conditions can be moved to a separate function. It will be much easier to read and maintain.

Reply:
Hm, in theory we could skip this special handling for playwright. We'd just have to add labels for that image as well so we have consistency across all images.

Reply:
Yup, I'm looking into adding the correct labels for the playwright image now. Thought I could merge this in the mean time.

Reply:
Latest playwright image has its browser versions added as image labels.